aten/src/ATen/SparseGCSTensorImpl.h

```diff
@@ -0,0 +1,63 @@
+#pragma once
+
+#include <ATen/Tensor.h>
+#include <ATen/TensorUtils.h>
+#include <ATen/SparseTensorUtils.h>
+#include <ATen/SparseCOOTensorImpl.h>
+#include <c10/core/TensorImpl.h>
+#include <c10/util/Exception.h>
+
+namespace at {
```

Comment:
The top of this file is the perfect place to put a general description of what GCS is and how the representation works. This is especially important here because we're doing an unusual generalization of the normal format (CSR) and so most people aren't going to know what it is; indeed, I can't even *google* for GCS.